torch/testing/_internal/common_methods_invocations.py

```diff
@@ -359,6 +359,29 @@ def sample_inputs_tensor_split(op_info, device, dtype, requires_grad):
                         args=(torch.tensor([1, 2, 3]),),
                         kwargs=dict(dim=1)),)
 
+def sample_inputs_linalg_det(op_info, device, dtype, requires_grad):
+    kw = dict(device=device, dtype=dtype)
+    inputs = [
```

Comment:
Does it work for 0x0 and 0xSxS inputs? I think it should be included here at the end of list (putting it first might break "out" tests).

Reply:
good idea, I added them in daccdb33d6